lib/order.rb

```diff
@@ -0,0 +1,72 @@
+require './lib/menu.rb'
+
+class Order
+
+	attr_reader :basket
+
+
+	def initialize (menu = Menu.new)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)